app/questionPage.html

```diff
@@ -28,103 +28,140 @@
+                <div class="nav-buttons col-md-3">
                     <button type="button" class="btn btn-success overview">Overview</button>
                 </div>
-                <div class="col-md-2">
+                <div class="nav-buttons col-md-3">
+                    <input type="checkbox" id="flag-checkbox">
+                    <button type="button" id="flag-button" class="btn"><i class="fas fa-flag"></i></button>
+                </div>
+                <div class="nav-buttons col-md-3">
+                    <button type="button" class="btn btn-info next">Next</button>
                 </div>
             </div>
         </div>
     </div>
-    <div id="overview_page">
-        <h1>Test Overview</h1>
-        <table class="overview_page_table">
-            <thead>
+</div>
+<div id="overview_page">
+    <div class="container">
+        <div class="row">
+            <img class="logo" src="images/branding-icons/maydenacademy_logotype_05.18_web.svg">
+        </div>
+        <div class="row">
+            <div class="left">
+                <h4>Test Overview</h4>
+            </div>
+            <div class="col-md-2 right">
+                <h4 class="timer"></h4>
+            </div>
+        </div>
+        <div class="row">
+            <table class="overview_page_table">
+                <thead>
                 <tr>
-                    <th>Question Id</th>
+                    <th>Question Number</th>
                     <th>Question</th>
                     <th>Answered</th>
                     <th>Flagged</th>
+                    <th>Return to Question</th>
                 </tr>
-            </thead>
-            <tbody class="overview_table_body"></tbody>
-        </table>
+                </thead>
+                <tbody class="overview_table_body"></tbody>
+            </table>
+        </div>
         <footer class="overview_footer">
-            <div class="row">
-                <div class="col-xs-10">
+            <div class="row footer-row">
+                <div class="col-md-4">
                     <p>Clicking this button submits the test once and for all there is no turning back!</p>
                 </div>
-                <div class="col-xs-2">
-                    <button type="button" id="finish" class="btn btn-danger finish">Finish</button>
+            </div>
+            <div class="row footer-row">
+                <div class="col-md-3">
+                    <button type="button" id="finish" class="btn finish">Finish</button>
                 </div>
             </div>
         </footer>
     </div>
-    <div id="result_page">
-        <div class="container">
-            <div class="userMessage">
+</div>
+<div id="result_page">
+    <div class="container">
+        <div class="userMessage">
+        </div>
+        <div class="row">
+            <img class="logo" src="images/branding-icons/maydenacademy_logotype_05.18_web.svg">
```

Comment:
I checked s1t4 (which was merged yesterday) had the logo <img> directly inside the "container". We should have same structure for this in every page I think.